presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/api/ControlAPI.java

```diff
@@ -41,7 +40,7 @@
 @Singleton
 public class ControlAPI
 {
-    private static final Logger LOGGER = LogManager.getLogger(ControlAPI.class);
+    private static final Logger LOGGER = Logger.get(ControlAPI.class);
```

Comment:
Why you need it given you removed all usages of logger?
Also, ditto on removal in separate commit.